sdk/python/feast/infra/materialization/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .batch_materialization_engine import (
+    BatchMaterializationEngine,
+    MaterializationJob,
+    MaterializationTask,
+)
+from .local_engine import LocalMaterializationEngine
+
+__all__ = [
```

Comment:
are these lines necessary? wouldn't a blank `__init__.py` suffice? that's what we currently have in e.g. `infra/offline_stores`

Reply:
I want the import to be nicer. Having a blank init makes the import something like:
```
feast.infra.materialization.local_engine.LocalMaterializationEngine
```

This makes it 
```
feast.infra.materialization.LocalMaterializationEngine
```